### PR TITLE
Remove deprecated

### PR DESCRIPTION
**Pull request by kdv24**
Progress toward removing the ten components that are deprecated in the component-library.

Screen Shot 2019-05-23 at 4 44 47 PM

* Should these each have been their own PR?

### CODE REVIEW

**Comment by jaronheard** [comment]
> Should these each have been their own PR?

@kdv24 I think that doing this all as one PR is fine!

**Comment by kdv24** [comment]
### Components identified as Deprecated (√ once removed):
- [x] Chart.js
- [x] ChartData.js
- [ ] HeatMap.js - reverted with last commit b/c will be used going forward
- [x] HexOverlay.js
- [x] Media.js
- [x] Sankey.js
- [x] ShareCollection.js
- [ ] StoryFooter.js - used in StoryCard.js & CivicStoryCard
- [ ] StoryLink.js - used in StoryFooter.js
- [x] Tag.js

Okay, should be good to go now. @jaronheard will make issue cards for the StoryLink and StoryFooter components referenced above.

File: packages/component-library/stories/index.js [hunk 1]
```diff
@@ -15,8 +15,6 @@ import dataTable from "./DataTable.story";
 import dropdownStory from "./DropdownMenu.story";
 import gradientScaleStory from "./GradientScale.story";
 import headerStory from "./Header.story";
-import heatMapStory from "./HeatMap.story";
```
Comment (jaronheard) [inline packages/component-library/stories/index.js]: Keep this line

File: packages/component-library/stories/index.js [hunk 2]
```diff
@@ -135,8 +132,6 @@ storiesOf("Component Lib|Maps", module)
 baseMapStory();
 boundaryMapStory();
 civicSandboxMapStory();
-heatMapStory();
```
Comment (jaronheard) [inline packages/component-library/stories/index.js]: Keep this line

**Comment by mendozaline** [comment]
Since heat maps are now created within the Base Map component, the original Heat Map component can be removed. Specifically this file here: 
https://github.com/hackoregon/civic/blob/master/packages/component-library/src/HeatMap/HeatMap.js

Also line 63 in this file can be removed as well: 
https://github.com/hackoregon/civic/blob/master/packages/component-library/src/index.js#L63

The Heat Map story and the lines in ` stories/index.js ` should be kept as noted by Jaron above.

**Comment by kdv24** [comment]
@mendozaline because I hadn't realized someone else was working on the HeatMap and I neglected to pull again before today, I'm concerned that my more recent changes will push over the top of the changes you made to the Heat Map story itself.  I think it'd be worth copying what you did so it can be played back on top of mine.  I can do it myself, but I'd rather you got credit for the work you did!

File: packages/component-library/stories/index.js (hunk 1 above)
Comment (kdv24) [reply packages/component-library/stories/index.js]: added back in

File: packages/component-library/stories/index.js (hunk 2 above)
Comment (kdv24) [reply packages/component-library/stories/index.js]: yup. done. thx!

**Comment by kdv24** [comment]
Thanks @mendozaline and @jaronheard!  I think this can be merged now.

**Pull request merged**